src/Contact/DataValidatorContact.php

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
can you add a public getErrors function that will return the errors if the validation fails? Declare it also in the interface.